### ISSUE TITLE
Backup mariadb-pvc: Cannot open on file...

### ISSUE THREAD

**Issue by splattner** [vshn/k8up-tutorial#2]
I get the following when creating the backup unsing `k8up/backup.yaml`

```
Removing locks...
created new cache in /.cache/restic
successfully removed locks
Listing all pods with annotation appuio.ch/backupcommand in namespace default
Listing snapshots
snapshots command:
0 Snapshots
backing up...
Starting backup for folder mariadb-pvc
done: 0.00% 
error cannot  on file /data/mariadb-pvc/mysql
error cannot  on file /data/mariadb-pvc/performance_schema
error cannot  on file /data/mariadb-pvc/wordpress
error cannot open on file /data/mariadb-pvc/aria_log.00000001
error cannot open on file /data/mariadb-pvc/aria_log_control
error cannot open on file /data/mariadb-pvc/ib_buffer_pool
error cannot open on file /data/mariadb-pvc/ib_logfile0
error cannot open on file /data/mariadb-pvc/ib_logfile1
error cannot open on file /data/mariadb-pvc/ibdata1
error cannot open on file /data/mariadb-pvc/ibtmp1
error cannot open on file /data/mariadb-pvc/multi-master.info
error cannot  on file /data/mariadb-pvc/mysql
error cannot  on file /data/mariadb-pvc/performance_schema
error cannot open on file /data/mariadb-pvc/tc.log
error cannot  on file /data/mariadb-pvc/wordpress
backup finished! new files: 0 changed files: 0 bytes added: 356
Listing snapshots
snapshots command:
1 Snapshots
sending webhook Starting backup for folder wordpress-pvc
done: 0.00% 
backup finished! new files: 1956 changed files: 0 bytes added: 356
Listing snapshots
snapshots command:
2 Snapshots
sending webhook Listing snapshots
snapshots command:
2 Snapshots
Removing locks...
successfully removed locks
Listing snapshots
snapshots command:
2 Snapshots
Sending webhooks to : %  
```

using minikube with docker driver

**Comment by Kidswiss** [comment]
Hi @splattner 

Thanks for interest in the K8up tutorial. We already found that issue last Friday and opened an internal ticket to have a look at that.

The original version of this tutorial was written for an older version of k8up where the backup pod was executed as root. Since that we changed the user to a non root user. While this seems to be working on some PVC providers (Gluster, EFS) it doesn't seem to work with all. Like the local providers of minikube or k3d.

I'm doing some testing to see if executing it as root solves the issue.
